### DISCUSSION TITLE
Mp3 container == mp3 error | Container doesn't contain mp3 definition

### DISCUSSION THREAD

**Discussion by martingrgv** [Tyrrrz/YoutubeExplode#743] [Q&A] [answered]
```` IStreamInfo audioStreamInfo = streamManifest.GetAudioStreams().Where(s => s.Container == Container.Mp3).GetWithHighestBitrate(); ````
Here the lambda expression gives error because of Container that doesn't contain Mp3 nor Mp4

**Comment by Tyrrrz** [comment, marked as answer]
That's correct, YouTube doesn't provide mp3 streams
